lib/rubocop/cop/style/it_block_parameter.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for blocks with one argument where `it` block parameter can be used.
+      #
+      # It provides three `EnforcedStyle` options:
+      #
+      # 1. `allow_named_parameter` (default) ... Detects only numbered block parameters.
```

Comment:
Here we probably need a different name that more clearly reflects that we're enforcing the use of `it` over `_1`. I'm also wondering if we don't need a more generic `BlockParameter` cop, as we already have a couple of them and the interplay between them might become confusing for the end users.